event/event.go

```diff
@@ -11,8 +11,19 @@ import (
 	"net/http"
 	"time"
 
+	"github.com/aau-network-security/haaukins/lab"
+	"github.com/aau-network-security/haaukins/store"
+	"github.com/aau-network-security/haaukins/svcs/ctfd"
+	"github.com/aau-network-security/haaukins/svcs/guacamole"
+	"github.com/aau-network-security/haaukins/virtual/docker"
+	"github.com/aau-network-security/haaukins/virtual/vbox"
+	"github.com/rs/zerolog/log"
 	"io"
 	"sync"
+<<<<<<< HEAD
```

Comment:
These `<<<<`, `=====` and `>>>>>>` must be removed, as they are git specific

Reply:
Yeah.